src/style.css

```
@@ -0,0 +1,66 @@
+* {
+  margin: 0;
+  padding: 0;
+  text-decoration: none;
+  list-style: none;
+  box-sizing: border-box;
```

```suggestion
  box-sizing: border-box;
```
* selector use only box-sizing.

reset default styles only for tags which necessary